etc/google-cloud-storage-test.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Note: it overlaps with etc/rse-accounts.cfg.

Reply:
this is a separate file that gets loaded by the oauth2 provider. it can't have anything else in it.

Reply:
few various s3 clients get the same behavior with default configuration files but it's usually possible to pass the parameters as arguments. Is it a per account/project configuration ?

Reply:
it's the same rse-accounts.cfg with the s3 credentials...

Reply:
... but it can have several storage for S3.  BTW, I don't understand the link RSE-signed urls. How do we know that we can have a signed url for a RSE ? `rucio download` use case for example.

Reply:
Linking PFNs from an RSE with signatures is the next step. Safest way to me seems a RSE attribute require_url_signature=True/False. So list_replicas will then know to call sign_url(pfn).